iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -54,35 +77,84 @@ protected void writeRootElementAttributes(ShuntCompensator sc, VoltageLevel vl,
 
     @Override
     protected void writeSubElements(ShuntCompensator sc, VoltageLevel vl, NetworkXmlWriterContext context) throws XMLStreamException {
+        IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_2, context, () -> {
+            try {
+                writeModel(sc, context);
+            } catch (XMLStreamException e) {
+                throw new UncheckedXmlStreamException(e);
+            }
```

Comment:
To make the code simpler, maybe you should move the try catch in the writeModel method.